src/solc_0.7/common/Interfaces/GameToken.sol

```diff
@@ -0,0 +1,52 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.7.1;
+
+/// @title Interface for the Game token
+
+interface GameTokenInterface {
+    event AssetsAdded(uint256 indexed id, uint24[] list);
+    event AssetsRemoved(uint256 indexed id, uint256 numRemoved);
```

Comment:
To be updated I guess?
`event AssetsAdded(uint256 indexed id, uint256[] assets, uint256[] values);`
`event AssetsRemoved(uint256 indexed id, uint256[] assets, uint256[] values, address to);`

Reply:
yup.